### PR TITLE
Update project license field in appdata

### PR DESCRIPTION
**Pull request by salarua**

### CODE REVIEW

**Comment by warp-core** [comment]
@salarua it's generally useful to provide some explanation of what a PR is doing and why you think it's a good change, rather than leaving the description empty.

**Comment by salarua** [comment]
I think the appdata should be updated because the `<project-license>` tag is for code only.

File: io.github.endless_sky.endless_sky.appdata.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <summary xml:lang="de">Weltraumhandels und Kampfsimulator</summary>
   <summary xml:lang="fr">Jeu d'exploration et de combat dans l'espace</summary>
   <developer_name>Michael Zahniser</developer_name>
-  <project_license>GPL-3.0 AND GPL-2.0 AND CC-BY-SA-4.0 AND CC-BY-SA-3.0 AND CC-BY-4.0 AND CC-BY-3.0 AND CC-BY-2.0 AND CC0-1.0</project_license>
+  <project_license>GPL-2.0+</project_license>
```
Comment (oo13) [inline io.github.endless_sky.endless_sky.appdata.xml:10]:
```suggestion
  <project_license>GPL-3.0-or-later</project_license>
```
GPL v2.0 is not allowed.

**Review by oo13 — CHANGES_REQUESTED** [review]

File: io.github.endless_sky.endless_sky.appdata.xml (hunk 1 above)
Comment (salarua) [reply io.github.endless_sky.endless_sky.appdata.xml:10]: I saw GPL 2.0 in the license and thought that it applied to some parts of the code. I'll update my PR accordingly.

**Comment by Amazinite** [comment]
We squash merge everything, so the commits in each PR don't matter too much aside from giving people a nice summary of what you changed in each commit.

File: io.github.endless_sky.endless_sky.appdata.xml (hunk 1 above)
Comment (Amazinite) [reply io.github.endless_sky.endless_sky.appdata.xml]: The codebase is GPLv3+.
https://github.com/endless-sky/endless-sky/blob/355cdfd83a0d153bdfa9bcd8338047ffd5aeeede/source/main.cpp#L463
The GPLv2 in the copyright file is from a single sound asset.
https://github.com/endless-sky/endless-sky/blob/355cdfd83a0d153bdfa9bcd8338047ffd5aeeede/copyright#L498-L502

**Comment by Amazinite** [comment]
The PR that originally added this file was #1500. The description for that PR linked documentation that describes how the appdata file should be formatted.

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-project_license
> The license given in the project_license tag should be the ‘main’ license of the project. For a software project, this is typically the license for the code.

Our code license is GPLv3+, so this PR checks out.

**Pull request merged**